test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,34 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+
+    @Test
+    public void isAnyNull() {
+        // empty list
+        assertNotNull();
+
+        // all objects not null
+        assertNotNull("abc", "ab", "a");
+        assertNotNull(1, 2);
+
+        // some null objects
+        assertAreNull((Object) null);
+        assertAreNull("abc", null);
+        assertAreNull(null, "", "abc", "ABC");
+        assertAreNull("", "abc", "a", null);
+        assertAreNull(null, 1, Integer.valueOf(1));
+        assertAreNull(null, "a", "b", null);
+
+        // all objects null
+        assertAreNull(null, null, null);
+    }
+    
+    private void assertAreNull(Object... obj) {
+        assertTrue(Utils.isAnyNull(obj));
+    }
+    
+    private void assertNotNull(Object... obj) {
+        assertFalse(Utils.isAnyNull(obj));
+    }
```

Comment:
Recall that in class, we talked about why these helper methods `assertAreNull` and `assertNotNull` are unnecessary here. The pre-existing code extracted into helper methods to avoid repeating `Arrays.asList` everywhere, but this is not the case for `isAnyNull`. Do you know what we can do instead? :)